integration/stability/hypervisor_stability_kill_test.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 HyperHQ Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This test will kill a running container's
+# hypervisor, and see how we react to cleanup.
+
+set -e -x
```

Comment:
do we need the `set -x` ?

Reply:
Not really necessary but I want to keep it to catch errors as early as possible.

Reply:
are we launching the test with `chronic`? If so, the `-x` will get nicely hidden unless we fail.
nb. though, afaik we don't set `-x` for any other tests ...

Reply:
Fair enough, I've dropped it.